Makefile

```diff
@@ -295,6 +295,12 @@ push-latest: container-latest
 push-name:
 	@echo "pushed: $(IMAGE):$(ARCH)-$(VERSION)"
 
+release:
```

Comment:
Could you add `release` to the list of phony targets on the first line of the Makefile?

Reply:
We could add the targets for the kgctl binaries as dependencies of `release` so that we know that running `make release` will automatically build all the needed binaries. Plus this was you wouldn't have to explicitly build the binaries in the github actions file. WDYT? Maybe its not easy because of the multiple OSes

Reply:
> Could you add `release` to the list of phony targets on the first line of the Makefile?

yes, that should be a phony target